src/Stickman.dpr

```diff
@@ -6,7 +6,7 @@
 
 {$R stickman.RES}
 {$DEFINE force16bitindices} //ez hib�s, p�r helyen, ha nincs kipontozva, meg kell majd n�zni
-{$DEFINE undebug} //Remove dot for release, add dot for dev
+{.$DEFINE undebug} //Remove dot for release, add dot for dev

```

Comment:
Azottmaradt, bár nem is emlékszem, hogy ezt csak RC mellett szoktuk-e kivenni.

Reply:
Developon release utan vissza szoktam rakni egy 'Return to dev' commitban. Rossz commitbol inditottad az elso branchet.